test/puppetlabs/puppetdb/integration/install_gems.clj

```diff
@@ -0,0 +1,30 @@
+(ns puppetlabs.puppetdb.integration.install-gems
+  (:require [clojure.java.io :as io]
+            [puppetlabs.puppetserver.cli.subcommand :as cli]
+            [puppetlabs.services.jruby.jruby-puppet-core :as jruby-puppet-core]
+            [puppetlabs.services.jruby-pool-manager.jruby-core :as jruby-core]
+            [clojure.string :as string]
+            [environ.core :refer [env]]))
```

Comment:
Presumably all of these sub-execs are bottoming out in direct (non-shell) execs -- if we're not sure, might want to verify.